app/services/CapitalGainsService.scala

```diff
@@ -40,7 +40,7 @@ trait CapitalGainsService {
   private[services] def capitalGains(atsData: AtsData): CapitalGains = {
       val capitalGainsData: DataHolder = atsData.capital_gains_data.get
       CapitalGains(atsData.taxYear,
-        atsData.utr.get,
+        atsData.utr.getOrElse(""),
```

Comment:
Seems to suggest an `Option` again?

Reply:
Same issue. For SA ATS this field is compulsory... Now that the viewModels are playing double duty then yes Option is the way to go. Thx.